WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:46 (_mysql_exceptions.OperationalError) (1054, "Unknown column 'username' in 'field list'") [SQL: u'INSERT INTO users (create_time, update_time, username, password_hash, email, is_activate) VALUES (%s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 11, 14, 40, 21, 427615), datetime.datetime(2017, 12, 11, 14, 40, 21, 427625), '123', 'pbkdf2:sha256:50000$dVFBPJtm$a7e073e313a3ee7901028f268d8b7752f3e9e8460eb0a9d4caeed8851285e6d8', '123@163.com', 0)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:80 Signature 'i1BvxXmq_d-XuVbGPBAGasvTbf8PeZYf5kQwMLhjrq8' does not match
ERROR passport.py:80 Signature 'i1BvxXmq_d-XuVbGPBAGasvTbf8PeZYf5kQwMLhjrq8' does not match
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:82 Signature 'i1BvxXmq_d-XuVbGPBAGasvTbf8PeZYf5kQwMLhjrq8' does not match
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:80 Signature '4b3y9uv9D3b4BijdR88xFVWFksOWvKEWKDTN8IaSAaQ' does not match
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:98 Class 'flask_sqlalchemy.BaseQuery' is not mapped
ERROR passport.py:98 Class 'flask_sqlalchemy.BaseQuery' is not mapped
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:100 Class 'flask_sqlalchemy.BaseQuery' is not mapped
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:83 Signature '5w34BB0b6v1lCtatV0w-pRlSljRCAp5hHYui0KFfyD0' does not match
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:50 (_mysql_exceptions.IntegrityError) (1062, "Duplicate entry '12345678@163.com' for key 'email'") [SQL: u'INSERT INTO users (create_time, update_time, username, password_hash, email, is_activate) VALUES (%s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 11, 15, 38, 52, 39993), datetime.datetime(2017, 12, 11, 15, 38, 52, 40008), '12345678', 'pbkdf2:sha256:50000$UkVhO8cG$a42d67b45b43e7e02d5da2fe476a79c79514e1b44f9540620e76d2ab7144dd00', '12345678@163.com', 0)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:78 (_mysql_exceptions.IntegrityError) (1062, "Duplicate entry '1234512' for key 'username'") [SQL: u'INSERT INTO users (create_time, update_time, username, password_hash, email, is_activate) VALUES (%s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 11, 16, 31, 11, 753082), datetime.datetime(2017, 12, 11, 16, 31, 11, 753092), '1234512', 'pbkdf2:sha256:50000$AmM5Uqc1$34d151de71e90bb27399f2aa086167a93389d0682c6de41a224d5cac13fcda8f', 'youthzhulingyun@163.com', 0)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp5,D9GowACXXfhaRy5aawAUAQ--.50S2 1512982371,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.190.127&hostid=smtp5&time=1512982371')
ERROR passport.py:78 (_mysql_exceptions.IntegrityError) (1062, "Duplicate entry 'flavio' for key 'username'") [SQL: u'INSERT INTO users (create_time, update_time, username, password_hash, email, is_activate) VALUES (%s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 11, 16, 54, 36, 584781), datetime.datetime(2017, 12, 11, 16, 54, 36, 584793), 'flavio', 'pbkdf2:sha256:50000$i2hBXeQV$e9442898ed44f06bc363c4bce7c192a79ea2a3657eaf5677aee442d470ceaddc', 'youthzhulingyun@163.com', 0)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:116 Signature expired
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:118 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:118 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:118 Signature expired
ERROR passport.py:118 Signature expired
ERROR passport.py:118 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:119 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:119 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:119 Signature expired
WARNING _internal.py:87  * Debugger is active!
ERROR passport.py:119 Signature '7eOymVfNtwjbvWxnZ5jlHDgomZbf7QJzaN5bTlKTaTU' does not match
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp1,C9GowACXpwuYkS5aOo4TAA--.2943S2 1513001369,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.190.127&hostid=smtp1&time=1513001369')
ERROR passport.py:84 (_mysql_exceptions.IntegrityError) (1062, "Duplicate entry 'youthzhulingyun@163.com' for key 'email'") [SQL: u'INSERT INTO users (create_time, update_time, username, password_hash, email, is_activate) VALUES (%s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 11, 22, 11, 39, 809322), datetime.datetime(2017, 12, 11, 22, 11, 39, 809343), 'flavio1', 'pbkdf2:sha256:50000$gutv9mk9$98536ae6fc799fc08f250e275b088ee4701bd27df3bedafaffe6dd32c13c2186', 'youthzhulingyun@163.com', 0)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp3,DdGowADHXnOmmC5aRjYlAA--.2463S2 1513003175,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.190.127&hostid=smtp3&time=1513003175')
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp3,DdGowAAXrnJymS5aWKIlAA--.2753S2 1513003378,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.190.127&hostid=smtp3&time=1513003378')
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp3,DdGowABnNk9cmi5aAScmAA--.2825S2 1513003614,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.186.27&hostid=smtp3&time=1513003614')
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR send_email.py:14 (554, 'DT:SPM 163 smtp3,DdGowABXqHz4mi5a7HMmAA--.2783S2 1513003768,please see http://mail.163.com/help/help_spam_16.htm?ip=111.199.190.127&hostid=smtp3&time=1513003768')
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR theatrical_film.py:184 (_mysql_exceptions.IntegrityError) (1364, "Field 'id' doesn't have a default value") [SQL: u'INSERT INTO theatrical_films_comment (create_time, update_time, user_name, comment, theatrical_film_id) VALUES (%s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 13, 15, 9, 27, 284458), datetime.datetime(2017, 12, 13, 15, 9, 27, 284472), 'flavio', 'test3', u'1')]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR theatrical_film.py:184 (_mysql_exceptions.IntegrityError) (1364, "Field 'id' doesn't have a default value") [SQL: u'INSERT INTO theatrical_films_comment (create_time, update_time, user_name, comment, theatrical_film_id) VALUES (%s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2017, 12, 13, 15, 12, 17, 963085), datetime.datetime(2017, 12, 13, 15, 12, 17, 963099), 'flavio', '123', u'1')]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:58 'BaseQuery' object has no attribute 'query'
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:58 unsupported operand type(s) for -: 'str' and 'int'
ERROR tvshow.py:58 unsupported operand type(s) for -: 'str' and 'int'
ERROR tvshow.py:58 unsupported operand type(s) for -: 'str' and 'int'
ERROR tvshow.py:58 unsupported operand type(s) for -: 'str' and 'int'
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:36 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:36 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:80 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:36 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:80 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:36 cannot concatenate 'str' and 'int' objects
ERROR tvshow.py:80 cannot concatenate 'str' and 'int' objects
ERROR tvshow.py:36 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:80 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:36 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:80 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
ERROR tvshow.py:89 hset() takes exactly 4 arguments (6 given)
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:123 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:162 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:161 'NoneType' object has no attribute 'url'
ERROR tvshow.py:122 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:159 'NoneType' object has no attribute 'get_real_url'
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:167 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR variety.py:50 (_mysql_exceptions.ProgrammingError) (1146, "Table 'qtfy.varietys' doesn't exist") [SQL: u'SELECT varietys.create_time AS varietys_create_time, varietys.update_time AS varietys_update_time, varietys.id AS varietys_id, varietys.name AS varietys_name, varietys.profile AS varietys_profile, varietys.type AS varietys_type, varietys.actor AS varietys_actor, varietys.premiere AS varietys_premiere, varietys.default_image AS varietys_default_image, varietys.score AS varietys_score, varietys.is_delete AS varietys_is_delete \nFROM varietys ORDER BY varietys.create_time DESC \n LIMIT %s, %s'] [parameters: (0, 10)]
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR _internal.py:87 Error on request:
Traceback (most recent call last):
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 200, in execute
    write(data)
  File "/Users/chunhua/.virtualenvs/flask/lib/python2.7/site-packages/werkzeug/serving.py", line 168, in write
    self.send_header(key, value)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR variety.py:50 global name 'VarietyShow' is not defined
ERROR variety.py:50 global name 'VarietyShow' is not defined
ERROR variety.py:50 global name 'VarietyShow' is not defined
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:149 Entity '<class 'app.models.VarietyNum'>' has no property 'tv_id'
WARNING _internal.py:87  * Debugger is active!
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:167 Entity '<class 'app.models.VarietyNum'>' has no property 'tv_id'
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:59 (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")
ERROR variety.py:50 (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:120 coercing to Unicode: need string or buffer, int found
ERROR tvshow.py:169 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:110 coercing to Unicode: need string or buffer, int found
ERROR variety.py:159 coercing to Unicode: need string or buffer, int found
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
